techhive/server.js

```diff
@@ -141,14 +144,75 @@ app.get('/users/:id', async (req, res) => {
   }
 });
 
+// Setup multer for profile picture uploads
```

Comment:
we already have multer setup above (line 24) for file uploads. 
in the future, can we reuse that code for profile pic uploads?